catalog/plugin/catalog-plugin-videothumbnail/src/main/java/org/codice/ddf/catalog/content/plugin/video/VideoThumbnailPlugin.java

```diff
@@ -490,12 +498,13 @@ private void deleteImageFiles() {
 
     imageFiles.add(new File(getGifFilePath()));
 
-    imageFiles.forEach(
```

Comment:
Sonarqube wanted to use `java.nio.Files.delete` instead of `java.io.File.delete`.

`java.io.File.delete` returns a Boolean indicating success, whereas `java.nio.Files.delete` throws an exception if the operation failed. So rather than testing if a file exists and trying to delete it, you can just try to delete it.